### PR TITLE
SEPA Remove Configuration and Set Default MandateType

### PR DESCRIPTION
**Pull request by sarahkoop**
### Summary of changes

 - Set default mandate type to ONE_OFF to align with iOS
 - Remove configuration from create mandate request and let gateway handle defaulting merchant account

 ### Checklist

 - ~[ ] Added a changelog entry~

- @sarahkoop

### CODE REVIEW

File: SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java [hunk 1]
```diff
@@ -75,52 +75,42 @@ public void setListener(SEPADirectDebitListener listener) {
...
-                    braintreeClient.sendAnalyticsEvent("sepa-direct-debit.configuration.failure");
-                    listener.onSEPADirectDebitFailure(configError);
+                    } else if (result.getApprovalUrl().equals("null")) {
+                        braintreeClient.sendAnalyticsEvent("sepa-direct-debit.create-mandate.success");
+                        // Mandate has already been approved
+                        braintreeClient.sendAnalyticsEvent("sepa-direct-debit.tokenize.requested");
+                        sepaDirectDebitApi.tokenize(result.getIbanLastFour(), result.getCustomerId(), result.getBankReferenceToken(), result.getMandateType().toString(), new SEPADirectDebitTokenizeCallback() {
+                                                                                                                                                                      @Override
+                                                                                                                                                                      public void onResult(@Nullable SEPADirectDebitNonce sepaDirectDebitNonce, @Nullable Exception tokenizeError) {
```
Comment (jaxdesmarais) [inline SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java:97]: nit: remove extra space

**Review by jaxdesmarais — APPROVED** [review]
couple of _very small_ cleanup comments but otherwise looks great!

**Review by sshropshire — APPROVED** [review]
🌊

File: SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java [hunk 2]
```diff
@@ -75,52 +75,42 @@ public void setListener(SEPADirectDebitListener listener) {
...
+                                if (sepaDirectDebitNonce != null) {
+                                    braintreeClient.sendAnalyticsEvent("sepa-direct-debit.tokenize.success");
+                                    listener.onSEPADirectDebitSuccess(sepaDirectDebitNonce);
+                                } else if (tokenizeError != null) {
+                                    braintreeClient.sendAnalyticsEvent("sepa-direct-debit.tokenize.failure");
+                                    listener.onSEPADirectDebitFailure(tokenizeError);
+                                }
+                }
```
[the remark this answers is not in the document]
Comment (sarahkoop) [reply SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java]: Yeah that was weird formatting - I used auto indent to fix

**Pull request merged**